server/fleet/service.go

```diff
@@ -686,6 +686,8 @@ type Service interface {
 
 	DeleteMDMAppleBootstrapPackage(ctx context.Context, teamID uint) error
 
+	GetMDMAppleBootstrapPackageSummary(ctx context.Context, teamID *uint) (MDMAppleHostStatusSummary, error)
```

Comment:
Same comment here regarding standardizing the signatures (return pointer vs struct).